test/test.js

```diff
@@ -32,3 +34,24 @@ describe('Output file', () => {
     console.dir(weaponData[0]);
   });
 });
+describe('Output file', () => {
```

Comment:
differentiate the describe. If needed, modify the other as well to specify the `Warframe Output file` or `Weapons Output File`